### ISSUE TITLE
Room-level default Sender Display property

### ISSUE THREAD

**Issue by wmwsagara** [matrix-org/matrix-spec#1392]

**Comment by turt2live** [comment]
Note that visual-only changes like this are unlikely to be successful because they do not account for actors who *will* try to figure out who sent a given message. Technical solutions which mimic Telegram's channels/announcements feature would be good I think.

**Comment by wmwsagara** [comment]
Yes, my proposal is visual only, just not to clutter with optional info. Some posters may not like to imply a personal responsibility prominently too. I did not intend to hide the sender. Users, may use options to see the source to see who sent it. That's fine with me. 

Here is similar rendering what I propose:
![Screenshot from 2023-01-04 13-45-28](https://user-images.githubusercontent.com/94213268/210493180-e271706d-9ed4-4062-9e23-2e2919f5fbd2.png)

**Comment by wmwsagara** [comment]
@turt2live, could you drop here a screenshot of Telegram's channels/announcements feature?

**Comment by turt2live** [comment]
The privacy of the sender would be more important for us at the spec level. At the client level, many clients wouldn't implement a feature where it's possible to just view the source to see who sent it. 

I don't have a screenshot of Telegram on hand, sorry.

**Comment by wmwsagara** [comment]
|The privacy of the sender would be more important for us at the spec level.

This is exactly our concern too. But the current implementation is otherwise, shows the photo of sender together with the name in the most prominent way :)

There are, at minimum, two types of rooms:
1. Room require to display the individual sender (human or bot). Eg. Group chat rooms. That is, every direct message to the room should be displayed as from the individual sender. That is, Matrix server includes the individual sender in the message to the room (eg. m.room.message).

2. Room require either not to display the sender or if prefer to display the sender, display only as the room name, not as the individual sender. Eg. World news room. That is, every direct message to the room should be displayed as from the room. Avatar is, room avatar. Name is, room name. That is, Matrix server includes the room address/room id as the sender in the message to the room (eg. m.room.message), thereby, effectively hiding the individual poster.

3. Reply to a message requires to be displayed the individual commenter.

Role of the room admin may not be able to avoid unless create rooms of different types, which is too restrictive.

Therefore, room admin has to choose the room message poster as none, individual or room.

I like to see the opinions of others regarding this on how best we could achieve this.
